frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/EditTaskId/EditTaskId.test.tsx

```diff
@@ -32,12 +24,24 @@ jest.mock('../../../../contexts/BpmnConfigPanelContext', () => ({
   metaDataFormRef: { current: undefined },
 });
 
+jest.mock('../../../../utils/bpmn/StudioModeler', () => {
+  return {
+    StudioModeler: jest.fn().mockImplementation(() => {
+      return {
+        getAllTasksByType: jest
+          .fn()
+          .mockReturnValue([{ id: 'Task_2' }, { id: 'task2' }, { id: 'task3' }]),
```

Comment:
Out of curiosity, why is it not `[{ id: 'task1'}, { id: 'task2'}, { id: 'task3'}`? 😊